packages/renderer/src/lib/feedback/SendFeedback.svelte

```diff
@@ -54,11 +56,10 @@ function handleUpdate(e: boolean): void {
 
 {#if displayModal}
 <div class='z-40'>
-  <Modal name="Share your feedback" on:close={hideModal}>
+  <Modal name="Share your feedback" on:close={(): Promise<void> => hideModal()}>
```

Comment:
We can't just provide hideModal as its signature require a boolean as first argument, and this missmatch with the on:close Event type.